deeppixel/iqa/brisque.py

```diff
@@ -0,0 +1,240 @@
+import collections
+from itertools import chain
+import urllib.request as request
+import pickle 
+import numpy as np
+import scipy.signal as signal
+import scipy.special as special
+import scipy.optimize as optimize
+import matplotlib.pyplot as plt
+import skimage
+from skimage import io,transform
+import cv2
+import libsvm
+from libsvm import svmutil
+
+def normalized(kernel):
+    return kernel / np.sum(kernel)
+
+
+def gaussian_kernel2d(n, SIGMA):
+    Y, X = np.indices((n, n)) - int(n/2)
```

Comment:
Can you please put docstrings for each of the functions as it is crucial for documentation